.github/dependabot.yml

```diff
@@ -12,6 +12,9 @@ updates:
     labels:
       - "CherryPick"
       - "dependencies"
+      - "6.13.z"
```

Comment:
Can we add 6.14.z here too?

Reply:
I think we might not require this here. because we do not want the PR's for this non-default(non master) branches

Reply:
I will remove these branches from here and add 6.14 in the master

Reply:
That needs to be added in the master